static/app/utils/withIssueTags.tsx

```diff
@@ -96,12 +91,7 @@ function withIssueTags<Props extends WithIssueTagsProps>(
     // Listen to tag store updates and cleanup listener on unmount
     useEffect(() => {
       const unsubscribeTags = TagStore.listen((storeTags: TagCollection) => {
-        const tags = assign(
-          {},
-          storeTags,
```

Comment:
It looks like `storeTags` used to be earlier in the assign sequence, just want to make sure that moving it to the end was purposeful

Reply:
looks like we don't even need `storeTags` actually, as the updated state should be included in `getIssueTags` already

Reply:
Perfect, I like the removal. makes more sense now